network-api/networkapi/buyersguide/templates/fragments/seal_of_approval.html

```diff
@@ -1,8 +1,10 @@
+{% load i18n %}
+
 {% if product.meets_minimum_security_standards is True %}
 <img
   class="seal-of-approval"
   src="{% if full_badge is True %}/_images/buyers-guide/badge-star.svg{% else %}/_images/buyers-guide/badge-star-mini.svg{% endif %}"
-  title="this {% if is_related_product is True %}related {% endif %}product meets our minimum security standards"
-  alt="this {% if is_related_product is True %}related {% endif %}product meets our minimum security standards"
+  title="{% if is_related_product is True %}{% trans 'this related product meets our minimum security standards' %}{% else %}{% trans 'this product meets our minimum security standards' %}{% endif %}"
```

Comment:
let's use double quotes here, to match the rest of the code.

Reply:
Fixed!